### ISSUE TITLE
Improve image fill array loop

### ISSUE THREAD

**Issue by enrichman** [enrichman/stegosecrets#30]
If the images are less than the parts we reuse the same images, i.e. if `p=5` and the images are 3 `[ image1.jpg, image2.jpg, image3.jpg ]` then an array with the same images will be created: `[ image1.jpg, image2.jpg, image3.jpg, image1.jpg, image2.jpg ]`.

The algorithm to do this is not "efficient" and can improved. It adds all the available images until the array is big enough, and then, if bigger, it will be truncated.

https://github.com/enrichman/stegosecrets/blob/1fcc29f07b68b648f511208d1da574c81438d59e/internal/encrypt/encrypt.go#L183-L193

Basically if 255 parts are required, and 254 images are available, in order to add just 1 image an array of 254+254 elements will be created, and then cropped to 255.
